staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -379,6 +385,16 @@ func NewConfig(codecs serializer.CodecFactory) *Config {
 		APIServerID:           id,
 		StorageVersionManager: storageversion.NewDefaultManager(),
 		TracerProvider:        tracing.NewNoopTracerProvider(),
+
+		// Default is to advertise /discovery endpoint if the feature is enabled
```

Comment:
What is the `/discovery` endpoint?! :D

Reply:
Actually, it looks like you have something duplicated here in the comments, which is confusing.

Reply:
rebase somehow caused the old comments to pop up again, fixed.